.changeset/popular-rules-divide.md

```diff
@@ -0,0 +1,16 @@
+---
+'@astrojs/rss': minor
+---
+
+Added `trailingSlash` option, to control whether the emitted URLs should have the trailing slash.
```

Comment:
Do we have a precedent for verb tense on these? I haven't been picky, but I've now seen the equivalent of:
- Added
- Add
- Adds
in recent PRs, and it would be nice if we had a standard! Anyone have any thoughts on this?

(Would also make "slashes" plural to agree with plural URLs)

```suggestion
Added `trailingSlash` option to control whether or not the emitted URLs should have trailing slashes.
```

AND in line 16 below: change to "won't have trailing slashes"